lib/src/buttons/loading_button.dart

```diff
@@ -27,6 +28,9 @@ class MUILoadingButton extends StatefulWidget {
   /// Text for Loading Button
   final String text;
 
+  // optional boxshadow for button
+  final bool shadow;
+
```

Comment:
Instead of having bool, please have List<BoxShadow> so optionally users can pass shadow if there is mismatch to the theme.

and default value of that parameter in constructor is
```dart
<BoxShadow>[
    BoxShadow(
        color: Color.fromRGBO(72, 76, 82, 0.16),
        offset: Offset(0, 12),
        blurRadius: 16.0,
    ),
]
```
so if no one pass the shadow default will be applied and in case someone pass the shadow while calling button default will get override.

Reply:
done